### DISCUSSION TITLE
Debug resolver not used

### DISCUSSION THREAD

**Discussion by fain182** [nuwave/lighthouse#2527] [Q&A] [answered]
Hello,
I added a resolver to the field of a Type:
```
type VideoContent implements Content {
....
    "Destination link"
    url: URL
    @field(resolver: "App\\GraphQL\\Queries\\Content\\TemporaryUrl")
....
}
```

but it seems that is ignored, there is some way to understand which resolvers are used? Or maybe it can be overwritten somewhere else?

Thanks!

**Comment by spawnia** [comment]
Does your query return other fields of `type VideoContent`? What does the field `url` return?

**Reply by fain182** [reply, marked as answer]
The URL is a just string with some validation...
Anyway it was a cache problem, but having something like "print-schema" command that printed the resolver for each field would have been useful to understand where the problem was...

Thank you for the help :slightly_smiling_face:
